packages/subgraph/schema.graphql

```diff
@@ -8,6 +8,8 @@ type Organization @entity {
   proposals: [Proposal!] @derivedFrom(field: "organization")
   supporterCount: Int!
   honeyLiquidity: BigInt
+  incentivisedPriceOracle: Bytes
+  unipool: Bytes
```

Comment:
Should we force them to not be null? `Bytes!`

Reply:
No, this is fine. We handle other addresses this way.